### ISSUE TITLE
Add "Execute as admin", "Apps settings" and "Uninstall" entry in the context menu of modern apps.

### ISSUE THREAD

**Issue by ale5000-git** [Open-Shell/Open-Shell-Menu#489]
**Describe the solution you'd like**
Please add "Execute as admin", "Apps settings" and "Uninstall" entry in the context menu of modern apps.

First, fourth and seventh elements in the screenshot.
See:
![modern-apps-context-menu](https://user-images.githubusercontent.com/15793015/95018939-df6daa80-0662-11eb-96dc-64963a50b74a.png)

**Comment by ge0rdi** [comment]
> Please add "Apps settings" and "Uninstall" entry in the context menu of modern apps.

`Uninstall` option should be already there.
Just try on apps in `Apps` folder.

> Fourth and seventh elements in the screenshot.

Would be much easier to post screenshots in English (the language that maintainers understand).

**Comment by ale5000-git** [comment]
Hi,
I expect to see it also on "searched" apps if possible.

I have found an image on Google:
![alt text](https://filestore.community.support.microsoft.com/api/images/2ff7c851-136c-4751-a8a2-21c4babe9c57)

**Comment by ge0rdi** [comment]
> I expect to see it also on "searched" apps if possible.

Yup. That would be really great and I'd like to have it as well.
But currently `Open-Shell` doesn't support jump-lists in search results.
I did have a look at that few times, but it doesn't seem to be quite easy to add that support :(

So maybe some day ...
